Components/ListLogos/_style.scss

```diff
@@ -0,0 +1,100 @@
+[is='flynt-list-logos'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
+
+  .container {
+    @include center;
+  }
+
+  .listLogos {
+    display: block;
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 24px;
+    text-align: center;
+
+    @media (min-width: $breakpoint-desktop) {
+      margin-bottom: 3rem;
+    }
+  }
+
+  .list {
+    @include grid(2, center, 12px, 12px);
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    @media (min-width: 375px) {
+      @include grid(3, center,16px, 16px);
+    }
+
+    @media (min-width: 576px) {
+      @include grid(4, center, 16px, 16px);
+    }
+
+    @media (min-width: $breakpoint-tablet) {
+      @include grid(6, center, 16px, 16px);
+    }
+
+    @media (min-width: $breakpoint-desktop) {
+      @include grid(8, center, 16px, 16px);
+    }
+  }
+
+  .media {
+    display: block;
+    height: 0;
+    padding-bottom: 70%;
+    position: relative;
+
+    @at-root .flyntComponent--card#{&} {
+      background-color: #fff;
```

Comment:
color could be background color variable

Reply:
i added the background-light variable for the default theme ... the others have to be in any case white white white or #fff #fff #fff : > // i'm btw not able to chain the @at-root selector with a comma but i'm also not hurt by it.